tox.ini

```diff
@@ -10,6 +10,5 @@ deps =
     nose==1.3.7
     webassets==dev
```

Comment:
Note that I couldn't run the tests with this line as is I had to switch it to `webassets==0.11.1`, the latest public release.